gyrokinetic/apps/gk_field.c

```diff
@@ -49,6 +50,52 @@ gk_field_calc_energy_dt(gkyl_gyrokinetic_app *app, const struct gk_field *field,
   field->calc_energy_dt_func(app, field, dt, energy_reduced);
 }
 
+void
+gk_field_calc_phi_wall(gkyl_gyrokinetic_app *app, struct gk_field *field, double tm)
```

Comment:
shouldn't this be in one of the sub gk_field files? as it only applies to some cases